package.json

```diff
@@ -12,7 +12,11 @@
         "lint": "npx eslint ./src/**/*.ts ./test/**/*.ts",
         "prebuild": "rimraf dist ethers",
         "build": "npm run build:contracts && npm run build:ts && copyfiles \"ethers/**/*.d.ts\" dist",
-        "build:ts": "bili",
+        "build:node": "npm run build:nats_node && bili --format cjs",
+        "build:front_end": "npm run build:nats_front_end && bili --format esm",
+        "build:nats_front_end": "grep -rl '\"nats.ws\"' ./src | xargs perl -i -pe's/nats\\.ws/nats\\.ws\\/lib\\/src\\/mod\\.js/g'",
+        "build:nats_node": "grep -rl '\"nats.ws/lib' ./src | xargs perl -i -pe's/nats\\.ws\\/lib\\/src\\/mod\\.js/nats\\.ws/g'",
+        "build:ts": "npm run build:front_end && npm run build:node",
```

Comment:
Could you please elaborate a bit on what these scripts does? Are `nats.ws` replaced by `nats.ws/lib/src/mod.js` for browser build and `nats.ws/lib/src/mod.js` replaced by `nats.ws` for Node.js?